packages/@react-spectrum/s2/stories/SelectBoxGroup.stories.tsx

```diff
@@ -0,0 +1,671 @@
+
+          {/* Text + Description */}
+          <div>
+            <h4 className={sectionHeadingStyles}>Text + Description</h4>
+            <SelectBoxGroup 
+              orientation="vertical" 
+              numColumns={1}
+              onSelectionChange={action('onSelectionChange')}>
+              <SelectBox value="text-desc">
+                <Text slot="label">Main Text</Text>
+                <Text slot="description">Additional description</Text>
+              </SelectBox>
+            </SelectBoxGroup>
+          </div>
+
+          {/* Illustration + Description */}
+          <div>
+            <h4 className={sectionHeadingStyles}>Illustration + Description</h4>
+            <SelectBoxGroup 
+              orientation="vertical" 
+              numColumns={1}
+              onSelectionChange={action('onSelectionChange')}>
+              <SelectBox value="illustration-desc">
+                <PaperAirplane />
+                <Text slot="description">Only description text</Text>
+              </SelectBox>
+            </SelectBoxGroup>
+          </div>
+
+          {/* Illustration + Text + Description */}
+          <div>
+            <h4 className={sectionHeadingStyles}>Illustration + Text + Description</h4>
+            <SelectBoxGroup 
+              orientation="vertical" 
+              numColumns={1}
+              onSelectionChange={action('onSelectionChange')}>
+              <SelectBox value="all-vertical">
+                <AlertNotice />
+                <Text slot="label">Full Vertical</Text>
+                <Text slot="description">Complete description</Text>
+              </SelectBox>
+            </SelectBoxGroup>
+          </div>
+
+        </div>
+      </div>
+
+      {/* Horizontal Orientation */}
+      <div>
+        <h3 className={subheadingStyles}>Horizontal Orientation</h3>
+        <div style={{display: 'flex', flexDirection: 'column', gap: 20}}>
+          
+          {/* Text Only */}
+          <div>
+            <h4 className={sectionHeadingStyles}>Text Only (Optimized)</h4>
+            <SelectBoxGroup 
+              orientation="horizontal" 
+              numColumns={1}
+              onSelectionChange={action('onSelectionChange')}>
+              <SelectBox value="h-text-only">
+                <Text slot="label">Simple Horizontal Text</Text>
+              </SelectBox>
+            </SelectBoxGroup>
+          </div>
+
+          {/* Illustration + Text */}
+          <div>
+            <h4 className={sectionHeadingStyles}>Illustration + Text</h4>
+            <SelectBoxGroup 
+              orientation="horizontal" 
+              numColumns={1}
+              onSelectionChange={action('onSelectionChange')}>
+              <SelectBox value="h-illustration-text">
+                <PaperAirplane />
+                <Text slot="label">Horizontal with Illustration</Text>
+              </SelectBox>
+            </SelectBoxGroup>
+          </div>
+
+          {/* Text + Description */}
```

Comment:
<img width="426" height="134" alt="Screenshot 2025-08-08 at 5 04 27 PM" src="https://github.com/user-attachments/assets/ab6e4756-0f87-471b-9c6b-67ac9017ba46" />
the text doesn't seem quite centered here